packages/models-library/src/models_library/api_schemas_webserver/catalog.py

```diff
@@ -137,50 +136,30 @@ class Config(OutputSchema.Config):
         ...
 
 
-class DEVServiceGet(ServiceGet):
-    # pylint: disable=too-many-ancestors
+_CATALOG_SERVICE_EXAMPLE: dict[str, Any] = deepcopy(
+    api_schemas_catalog_services.DEVServiceGet.Config.schema_extra["example"]
+)
+_CATALOG_SERVICE_EXAMPLE.update(
+    {
+        "inputs": {
+            f"input{i}": example
+            for i, example in enumerate(ServiceInputGet.Config.schema_extra["examples"])
+        },
+        "outputs": {"outFile": ServiceOutputGet.Config.schema_extra["example"]},
+    }
+)
```

Comment:
something like this might also work
```suggestion
_CATALOG_SERVICE_EXAMPLE: dict[str, Any] = {
    ***deepcopy(
        api_schemas_catalog_services.DEVServiceGet.Config.schema_extra["example"]
    ),
    "inputs": {
        f"input{i}": example
        for i, example in enumerate(ServiceInputGet.Config.schema_extra["examples"])
    },
    "outputs": {"outFile": ServiceOutputGet.Config.schema_extra["example"]},
})
```